third_party/2and3/pynamodb/__init__.pyi

```diff
@@ -0,0 +1 @@
+__license__ = ...  # type: str
```

Comment:
I don't know of a general tool, but `codemod.py ' = ...  # type: ' ': '` works pretty well for these specific ones.

We should probably update stubgen too.